### DISCUSSION TITLE
Mistype in template: 'use strict;' should be 'use strict';

### DISCUSSION THREAD

**Discussion by r-pankevicius** [RussCoder/djvujs#43] [General]
Hi, @RussCoder ,

take a look at line 7 of file:
https://github.com/RussCoder/djvujs/blob/master/library/rollup.config.js

intro: "function DjVuScript() {\n'use strict;'",
should be:
intro: "function DjVuScript() {\n'use strict';",

**Comment by RussCoder** [comment]
Hello! Yes, you are right.
Maybe it even a bit decreases the performance, since they say in the strict mode the code can be better optimized. 
Thanks for the mention. I will fix it.

**Comment by RussCoder** [comment]
Fixed in the last release (library's version 0.5.1)
